py/server/deephaven/updateby.py

```diff
@@ -267,3 +266,85 @@ def forward_fill(cols: Union[str, List[str]]) -> UpdateByOperation:
         return UpdateByOperation(j_updateby_op=_JUpdateByOperation.Fill(cols))
     except Exception as e:
         raise DHError(e, "failed to create a forward fill UpdateByOperation.") from e
+
+
+def rolling_sum_tick(cols: Union[str, List[str]], rev_ticks: int, fwd_ticks: int = 0) -> UpdateByOperation:
+    """Creates a rolling sum UpdateByOperation for the supplied column names, using ticks as the windowing unit. Ticks 
+    are row counts and you may specify the reverse and forward window in number of rows to include. The current row 
+    is considered to belong to the reverse window but not the forward window. Also, negative values are allowed and 
+    can be used to generate completely forward or completely reverse windows. 
+
+    Here are some examples of window values:
+        rev_ticks = 1, fwd_ticks = 0 - contains only the current row
+        rev_ticks = 10, fwd_ticks = 0 - contains 9 previous rows and the current row
+        rev_ticks = 0, fwd_ticks = 10 - contains the following 10 rows, excludes the current row
+        rev_ticks = 10, fwd_ticks = 10 - contains the previous 9 rows, the current row and the 10 rows following
+        rev_ticks = 10, fwd_ticks = -5 - contains 5 rows, beginning at 9 rows before, ending at 5 rows before  the
+            current row (inclusive)
+        rev_ticks = 11, fwd_ticks = -1 - contains 10 rows, beginning at 10 rows before, ending at 1 row before the
+            current row (inclusive)
+        rev_ticks = -5, fwd_ticks = 10 - contains 5 rows, beginning 5 rows following, ending at 10 rows  following the
+            current row (inclusive)
+
+
+    Args:
+        cols (Union[str, List[str]]): the column(s) to be operated on, can include expressions to rename the output,
+            i.e. "new_col = col"; when empty, update_by perform the forward fill operation on all columns.
+        rev_ticks (int): the look-behind window size (in rows/ticks)
+        fwd_ticks (int): the look-forward window size (int rows/ticks), default is 0
+
+    Returns:
+        an UpdateByOperation
+
+    Raises:
+        DHError
+    """
+    try:
+        cols = to_sequence(cols)
+        return UpdateByOperation(j_updateby_op=_JUpdateByOperation.RollingSum(rev_ticks, fwd_ticks, *cols))
+    except Exception as e:
+        raise DHError(e, "failed to create a rolling sum (tick) UpdateByOperation.") from e
+
+
+def rolling_sum_time(ts_col: str, cols: Union[str, List[str]], rev_time: Union[int, str],
+                     fwd_time: Union[int, str] = 0) -> UpdateByOperation:
+    """Creates a rolling sum UpdateByOperation for the supplied column names, using time as the windowing unit. This 
```

Comment:
I expect more users will use the string duration specifiers than nanoseconds and we should either supplement these examples with `hh:mm:ss` examples or use strings exclusively.